app/gui/view/src/component_browser.rs

```diff
@@ -112,7 +112,7 @@ impl component::Frp<Model> for Frp {
                 &size,
                 &gap,
                 |list_size, list_x, size, gap| {
-                    Vector2(*list_x - list_size.x / 6.0, - size.y / 2.0 - gap - NODE_HEIGHT / 2.0)
+                    Vector2(*list_x - list_size.x / 2.0, - size.y / 2.0 - gap - NODE_HEIGHT / 2.0)
```

Comment:
The `*list_x` could just be `list_x` (it was already like that, but as long as we're touching it, why not fix it).

It's not obvious what this math does. Either named subexpressions, or a comment, or maybe named constants could help the reader understand this calculation.

Reply:
Thanks, fixed. Also added more subexpressions.